topics/introduction/faqs/fetched_data.md

```diff
@@ -0,0 +1,8 @@
+---
+title: I fetched data from a remote website but now I’m logged out of Galaxy and my data is gone?
+area: recommended browser
+box_type: tip
+layout: faq
+---
+
+This is a known bug with Chrome + Galaxy, we’re working on it. For now we can recommend using Firefox (known to work) or trying another browser.
```

Comment:
Could you link to the issues in the text, maybe it helps motivate the developers to fix it https://github.com/galaxyproject/galaxy/issues/11374

Reply:
Sure! Doing that now.